### ISSUE TITLE
Warning: 'remote_user' as a role param has been deprecated

### ISSUE THREAD

**Issue by goetzc** [stackbuilders/sb-debian-base#70]
When running the role:
```
[DEPRECATION WARNING]: Using 'remote_user' as a role param has been deprecated. In the future, these values should be 
entered in the `vars:` section for roles, but for now we'll store it as both a param and an attribute..
This feature 
will be removed in a future release. Deprecation warnings can be disabled by setting deprecation_warnings=False in 
ansible.cfg.
```

**Comment by lpaulmp** [comment]
I think this is warning come from an external role.

**Comment by goetzc** [comment]
Some external roles do also have this issue, and the issue is in this role too:
`meta/main.yml`
`tasks/main.yml`

**Comment by goetzc** [comment]
Fixed with commit 391935ceae.
